packages/esm-patient-chart-app/src/config-schema.ts

```diff
@@ -32,45 +32,51 @@ export const esmPatientChartSchema = {
-    _description: 'Shows the All Encounters Tab of Patient Visits section in Patient Chart',
     _default: false,
-  },
-  startVisitLabel: {
-    _type: Type.String,
-    _description: 'Custom label to use on the start visit actions',
-    _default: '',
-  },
-  showServiceQueueFields: {
-    _type: Type.Boolean,
-    _description: 'Whether start visit form should display service queue fields`',
-    _default: false,
-  },
-  priorityConceptSetUuid: {
-    _type: Type.ConceptUuid,
-    _description: 'The UUID of the priorities for the queues.',
-    _default: '9e123c90-76ac-4eaa-8d40-35577781eb46',
-  },
-  defaultPriorityConceptUuid: {
-    _type: Type.ConceptUuid,
-    _description: 'The UUID of the default priority for the queues eg Not urgent.',
-    _default: '9e123c90-76ac-4eaa-8d40-35577781eb46',
-  },
-  serviceConceptSetUuid: {
-    _type: Type.ConceptUuid,
-    _description: 'The UUID of the services for the queues.',
-    _default: 'f9c6a3f8-a5d0-4034-ac77-e413c4b3c620',
-  },
-  statusConceptSetUuid: {
-    _type: Type.ConceptUuid,
-    _description: 'The UUID of the statuses for the queues.',
-    _default: '13c9b9fb-3396-4139-98e1-59938d544168',
-  },
-  defaultStatusConceptUuid: {
-    _type: Type.ConceptUuid,
-    _description: 'The UUID of the default status for the queues eg Waiting.',
-    _default: '136203AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+    _description: 'Disable notes/tests/medications/encounters tabs when empty',
+  },
+    showAllEncountersTab: {
+      _type: Type.Boolean,
+      _description: 'Shows the All Encounters Tab of Patient Visits section in Patient Chart',
+      _default: false,
+    },
+    startVisitLabel: {
+      _type: Type.String,
+      _description: 'Custom label to use on the start visit actions',
+      _default: '',
+    },
+    showServiceQueueFields: {
+      _type: Type.Boolean,
+      _description: 'Whether start visit form should display service queue fields`',
+      _default: false,
+    },
+    priorityConceptSetUuid: {
+      _type: Type.ConceptUuid,
+      _description: 'The UUID of the priorities for the queues.',
+      _default: '9e123c90-76ac-4eaa-8d40-35577781eb46',
+    },
+    defaultPriorityConceptUuid: {
+      _type: Type.ConceptUuid,
+      _description: 'The UUID of the default priority for the queues eg Not urgent.',
+      _default: '9e123c90-76ac-4eaa-8d40-35577781eb46',
+    },
+    serviceConceptSetUuid: {
+      _type: Type.ConceptUuid,
+      _description: 'The UUID of the services for the queues.',
+      _default: 'f9c6a3f8-a5d0-4034-ac77-e413c4b3c620',
+    },
+    statusConceptSetUuid: {
+      _type: Type.ConceptUuid,
+      _description: 'The UUID of the statuses for the queues.',
+      _default: '13c9b9fb-3396-4139-98e1-59938d544168',
+    },
+    defaultStatusConceptUuid: {
+      _type: Type.ConceptUuid,
+      _description: 'The UUID of the default status for the queues eg Waiting.',
+      _default: '136203AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+    },
   },
```

Comment:
Hey @icrc-jofrancisco, it appears this unused brace is causing Prettier to fail. Please remove it and commit again.

Reply:
Hi @denniskigen , Done.